cohere/responses/generation.py

```diff
@@ -114,3 +115,22 @@ def prompt(self) -> str:
         """Returns the prompt used as input"""
         return self[0].prompt  # should all be the same
 
+
+StreamingTokenLikelihood = NamedTuple("TokenLikelihood", [("index", int), ("token", str), ("likelihood", float)])
```

Comment:
we changed "token" with "text" - both returned right now, but "token" will disappear soon

Reply:
I saw your message, will fix everything up and complete this after #160 is merged.